woocommerce/woocommerce
# Update WCA deactivation hook to use WC_PLUGIN_FILE

### All Submissions:



### Changes proposed in this Pull Request:

1. Replaces `WC_ADMIN_PLUGIN_FILE` with `WC_PLUGIN_FILE` where the deactivation hook is used. 
2. Removed `DeactivatePlugin` note since WCA is now embedded in the core. 



Closes #32522 .

### How to test the changes in this Pull Request:

1. Install [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
2. Activate WooCommerce and navigate to `Tools -> Cron Events`. Search for `wc_admin_process_orders_milestone` event. 
3. Open a new tab and navigate to `Plugins -> Installed Plugins`
4. Deactivate WooCommerce and refresh `Tools -> Cron Events` tab.
5. Confirm `wc_admin_process_orders_milestone` is no longer exist

### Other information:





### FOR PR REVIEWER ONLY:
